### DISCUSSION TITLE
Another solution for NumberToString

### DISCUSSION THREAD

**Discussion by warny** [Humanizr/Humanizer#1528] [Ideas]
Hi,
Had not been told about your library, I made my own. 
I had a different approach relying heavily on configuration files. I suggest you take a look at my work to see if you could be interested in reusing it (you are free to do so). We can discuss the matter if you want.
I worked with BigInt, so i can convert numbers bigger than 2^64. My library is optimized for the Chuquet/Conway's system, both with short and long scale (or any scale you can imagine) : https://en.wikipedia.org/wiki/Names_of_large_numbers
Here's my class : https://github.com/warny/All-purpose-Utilities/blob/master/Utils/Mathematics/NumberToStringConverter.cs
Here's my configuration files : https://github.com/warny/All-purpose-Utilities/tree/master/Utils/Mathematics/NumberConverterConfigurations

Best regards,
Olivier MARTY

**Comment by aloisdg** [comment]
huitante is ch but no be
